src/inpod.rs

```diff
@@ -69,6 +69,7 @@ impl WorkloadUid {
 #[derive(Debug, Clone)]
 struct WorkloadInfo {
     workload_uid: WorkloadUid,
+    pod_info: Option<istio::zds::PodInfo>,
```

Comment:
Can we:

1. Move `workload_uid` to a top-level field in `WorkloadData`
2. Move `pod_info` to a top-level field in `WorkloadData`

drop `WorkloadInfo` -> workloaddata.workloadinfo.blah feels unnecessary.